cfme/tests/cloud/test_targeted_refresh.py

```diff
@@ -65,7 +122,24 @@ def test_ec2_targeted_refresh_network():
             2. Network UPDATE
             3. Network DELETE
     """
-    pass
+    # create
+    network = provider.mgmt.create_network()
+    if not network:
+        pytest.fail("Network wasn't successfully created using API!")
+    request.addfinalizer(lambda: cleanup_if_exists(network))
+    network_collection = appliance.rest_api.collections.cloud_networks
+    wait_for(lambda: network_collection.get(ems_ref=network.uuid), delay=DELAY, timeout=TIMEOUT,
+             handle_exception=True)
+
+    # update - change name
+    new_name = fauxfactory.gen_alpha()
```

Comment:
Thoughts on having a descriptive string like `test-network` before the random string? Just so we know they were created by tests in case we need to clean them up manually.

Reply:
Reworked.